bin/aapp_dr_runner.py

```diff
@@ -204,11 +199,7 @@ def cleanup_aapp_workdir(config):
     try:
         filelist = glob(
             '%s/*' % config['aapp_processes'][config.process_name]['working_dir'])
-        dummy = [os.remove(s) for s in filelist if os.path.isfile(s)]
-        #filelist = glob('%s/*' % self.working_dir)
-        # LOG.info("Number of items left after cleaning working dir = " +
-        #          str(len(filelist)))
-        #LOG.debug("Files: " + str(filelist))
+        _ = [os.remove(s) for s in filelist if os.path.isfile(s)]
```

Comment:
Same here, probably better written as a regular for loop block.

Reply:
Done, thx!